history-of-the-quakers-v4/original/11-book-7-chapter-5.adoc

```diff
@@ -480,12 +503,18 @@ They were all taken up by an attachment in November, 1721, and carried to Ailsbu
 On the 20th and 22nd of October, 1722,
```

Comment:
Also, on line 497, there's a name of someone and I"m not sure if it's Olliffe, or Ollisse.